src/Microsoft.IdentityModel.Tokens/Validation/Validators.Lifetime.cs

```diff
@@ -46,5 +66,142 @@ public static void ValidateLifetime(DateTime? notBefore, DateTime? expires, Secu
 
             ValidatorUtilities.ValidateLifetime(notBefore, expires, securityToken, validationParameters);
         }
+
+        /// <summary>
+        /// Validates the lifetime of a <see cref="SecurityToken"/>.
+        /// </summary>
+        /// <param name="notBefore">The 'notBefore' time found in the <see cref="SecurityToken"/>.</param>
+        /// <param name="expires">The 'expiration' time found in the <see cref="SecurityToken"/>.</param>
+        /// <param name="securityToken">The <see cref="SecurityToken"/> being validated.</param>
+        /// <param name="validationParameters"><see cref="TokenValidationParameters"/> required for validation.</param>
+        /// <param name="callContext"></param>
```

Comment:
this is empty

Reply:
Following up on our chat regarding all `CallContext` references.